src/functions-reference/hidden_markov_models.Rmd

```diff
@@ -0,0 +1,69 @@
+# Hidden Markov Models (with discrete hidden variables)
```

Comment:
Is there any way to avoid the parenthesis in the title?

Looks weird.

Discrete Hidden Markov Models?

Even just Hidden Markov Models seems fine to me.